lib/modules/manager/asdf/upgradeable-tooling.ts

```diff
@@ -220,6 +220,14 @@ export const upgradeableTooling: Record<string, ToolingDefinition> = {
       extractVersion: '^v(?<version>\\S+)',
     },
   },
+  gleam: {
+    asdfPluginUrl: 'https://github.com/asdf-community/asdf-gleam.git',
+    config: {
+      datasource: GithubTagsDatasource.id,
```

Comment:
test failed. should it use releases or tags?

Reply:
Lemme re-check. Thanks.

Reply:
Both are populated. I think I see an inconsistency, but why would it matter if they both have the same values (not that it wouldn't semantically, though)?

Reply:
35373e325

Reply:
This may be wrong, but I think quite a few asdf would be wrong too. Let's look another time